source/contribute/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Contribute
+github_link: contribute/index.html
+indexed: true
+---
+
+## Contribute to Shopware
+
+Shopware is available under dual license (AGPL v3 and proprietary license). If you want to contribute code (features or bugfixes), you have to create a pull request and include valid license information. You can either contribute your code under New BSD or MIT license.
+
+If you want to contribute to the backend part of Shopware, and your changes affect or are based on ExtJS code, they must be licensed under GPL V3, as per license requirements from Sencha Inc.
+
+If you are not sure which license to use, or want more details about available licensing or the contribution agreements we offer, you can contact us at <contact@shopware.com>.
+
+- **Contribution guide**: <https://github.com/shopware/shopware/blob/master/CONTRIBUTING.md>
+- **License**: Dual license AGPL v3 / Proprietary
+- **Github Repository**: <https://github.com/shopware/shopware>
+- **Issue Tracker**: <http://jira.shopware.de/jira>
+
+## Contribute to the Dev Docs
+
+These doc pages are also available on Github and open to contribution from the community. 
+
+- **Github Repository**: <https://github.com/shopware/devdocs>
+- **License**: Proprietary
+
+Contributions to this project should aim at improving the overall coverage / quality of the Shopware 5 documentation page.
+ 
+All contributions should respect the following styling guidelines.
+
+### Language
+
+All documents should be written in American or UK English. Should it be necessary, disambiguation between the two should be done on a per occurrence basis, giving preference to the one most commonly used on similar contexts (other documentation sites, for example)
```

Comment:
Regarding American vs. UK English: In my opinion we should go with only one of them.
I highly opt for American English because most programming languages/markups use american english name.
See CSS `color` for example.